ui/raidboss/emulator/data/network_log_converter/ParseLine.ts

```diff
@@ -23,13 +23,14 @@ import { LineEvent36 } from './LineEvent0x24';
 import { LineEvent37 } from './LineEvent0x25';
 import { LineEvent38 } from './LineEvent0x26';
 import { LineEvent39 } from './LineEvent0x27';
+import LogRepository from './LogRepository';
 
 export default class ParseLine {
-  static parse(repo, line) {
+  static parse(repo: LogRepository, line: string): LineEvent | false {
     let ret;
 
     const parts = line.split('|');
-    const event = parts[0];
+    const event = parts[0] ?? '';
```

Comment:
```suggestion
    const event = parts[0];
```

I think this isn't necessary.

Reply:
![image](https://user-images.githubusercontent.com/6119598/119269751-4a0a3100-bbc7-11eb-91ae-018cd1e1c27f.png)

Reply:
I guess this could be adjusted to:
```typescript
    const parts = line.split('|');
    const event = parts[0];

    // Don't parse raw network packet lines
    if (!event || event === '252')
      return;
```

Reply:
Oh fair, sorry! I like your adjustment that you suggest here.